pkg/core/archive_experiment.go

```diff
@@ -14,13 +14,56 @@
 package core
 
 import (
+	"context"
+	"time"
+
 	"github.com/jinzhu/gorm"
+
+	"github.com/pingcap/chaos-mesh/api/v1alpha1"
 )
 
 // ExperimentStore defines operations for working with archive experiments
-type ExperimentStore interface{}
+type ExperimentStore interface {
+	// List returns a archive experiment list from the datastore.
+	List(context.Context) ([]*ArchiveExperiment, error)
+
+	// ListByKind returns a archive experiment list by kind from the datastore.
+	ListByKind(context.Context, string) ([]*ArchiveExperiment, error)
+
+	// Find returns a archive experiment by ID.
```

Comment:
What is "ID"?  Do we assign a number to each archive?  
s/a archive/an archive/g

Reply:
If we store an experiment in DB, the id is necessary. It may be auto-increment.